woocommerce/woocommerce
# Adding basic Woo AI settings screen

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Adding basic settings page to store settings involved with AI features. The ones currently added are those anticipated to be capture during onboarding, as you can [see in the original issue](https://github.com/woocommerce/woocommerce/issues/38746#issuecomment-1597768556).

Closes #38746 



### Screenshots

![image](https://github.com/woocommerce/woocommerce/assets/444632/3c2c3bfb-e575-4f72-a4be-a3c29157651d)



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout branch
2. Activate WooAI plugin in your environment
3. Navigate to WooCommerce -> Settings -> Advanced -> Features
4. Screen should appear as in screenshot above.
5. Fields should be disabled when "Enable" checkbox in unchecked.
6. Tone description should update as you change the selection.
7. Update settings, and submit.
8. Settings should persist.
